docs/docs/manual/exporting.md

```diff
@@ -95,7 +95,7 @@ Anything that appears inside doubled curly braces ({{ }}) is treated as a GREL e
 ```
 
 :::caution
-Note that some syntax is different in this tool than elsewhere in OpenRefine: a forward slash must be escaped with a backslash, while other characters do not need escaping. You cannot, at this time, include a closing curly brace (}) anywhere in your expression, or it will cause it to malfunction.
+Note that some syntax is different in this tool than elsewhere in OpenRefine: a forward slash must be escaped with a backslash, while other characters do not need escaping. You cannot, at this time, include doubled closing curly brace (}}) anywhere in your expression, or it will cause it to malfunction.
```

Comment:
Now it should be possible to include two closing braces by escaping them, right? Using `\}\}` should work

Reply:
Yes, but I'm not sure how to explain it 😂. Are there any other expressions that may be related to this problem?

Reply:
Intuitively I think it should only be braces that need escaping, I cannot think of anything else. But do check with the code :)